### ISSUE TITLE
org.influxdb.InfluxDBException$FieldTypeConflictException

### ISSUE THREAD

**Issue by xuzikun2003** [lensesio/stream-reactor#733]
# Issue Guidelines

Please review these questions before submitting any issue?

### What version of the Stream Reactor are you reporting this issue for?
Latest version

### Are you running the correct version of Kafka/Confluent for the Stream reactor release?
Yes

### Do you have a supported version of the data source/sink .i.e Cassandra 3.0.9?
No

### Have you read the [docs](https://docs.lenses.io/connectors/)?
Yes

### What is the expected behaviour?
The connector should be able to cast values automatically.

### What was observed?

I am getting the following error when I turn on InfluxDB sink to write data to InfluxDB.
[2021-01-17 18:45:07,950] ERROR WorkerSinkTask{id=sparkceip-0} Task threw an uncaught and unrecoverable exception. Task is being killed and will not recover until manually restarted. Error: org.influxdb.InfluxDBException$FieldTypeConflictException: partial write: field type conflict: input field "value" on measurement "measureA" is type integer, already exists as type float dropped=85 (org.apache.kafka.connect.runtime.WorkerSinkTask:586)
java.lang.RuntimeException: org.influxdb.InfluxDBException$FieldTypeConflictException: partial write: field type conflict: input field "value" on measurement "measureA" is type integer, already exists as type float dropped=85

The KCQL query that I am using is
INSERT INTO measureA SELECT * FROM test

Basically the there is column named "value" in the measurement measureA. This column contains both float and integers. The float are inserted first and then later the int are inserted. However, the type conflict occurs when the int is inserted after the float.

Anything I can do to unblock myself.
